### DISCUSSION TITLE
How can i run windows apps in Sandboxie?

### DISCUSSION THREAD

**Discussion by Dark-Cod3r** [sandboxie-plus/Sandboxie#2318] [Q&A Feedback] [answered]
Hi. So i have installed Whatsapp from Microsoft Store. Now i want to run it from sandboxie as well (because i want to instances of that app). How can i do that.?! i am able to run software easily but apps installed from Microsoft store, unable to find an option to run them. 
is it possible..?!
Thank you

**Comment by isaak654** [comment, marked as answer]
There is no support for UWP apps at this time, so you may want to check #19 or the [Sandboxie roadmap](https://www.wilderssecurity.com/threads/sandboxie-roadmap.445545/) to stay updated with latest news.
